werf.sdk-playground.yaml

```
@@ -0,0 +1,18 @@
+image: sdk-playground
+dockerfile: ./Dockerfile.sdk-playground
+context: .
+contextAddFiles:
+  - packages/sdk-playground/dist
+  - packages/sdk-playground/.env
+---
+image: sdk-playground-nginx
+dockerfile: ./Dockerfile.sdk-playground-nginx
+context: .
+dependencies:
+- image: sdk-playground
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 2 but found 0

(indentation)